python/xorbits/_mars/dataframe/utils.py

```diff
@@ -1290,11 +1290,15 @@ def create_sa_connection(con, **kwargs):
         close = True
         dispose = False
     else:
-        engine = sa.create_engine(con, **kwargs)
-        con = engine.connect()
-        close = True
-        dispose = True
-
+        try:
+            engine = sa.create_engine(con, **kwargs)
+            con = engine.connect()
+            close = True
+            dispose = True
+        except:  # pragma: no cover
```

Comment:
Is it possible to narrow down the exception type? There could be other errors like network error which should not be captured.